src/main/scala/no/ndla/imageapi/ImageApiProperties.scala

```diff
@@ -46,7 +46,7 @@ object ImageApiProperties extends LazyLogging {
 
   val StorageName = s"$Environment.images.ndla"
 
-  val SearchIndex = "images"
+  lazy val SearchIndex = propOrElse("SEARCH_ALIAS", "images")
```

Comment:
@kstigen Har du en kommentar til dette, dette var jo opplegget du ønsket å innføre. Noen argumenter for mot?